irods_specific_implementation/CMakeLists.txt

```diff
@@ -169,6 +167,7 @@ foreach(API_NAME ${API_SERVER_NAMES})
                         ${IRODS_EXTERNALS_FULLPATH_JSON}/include
                         ${IRODS_EXTERNALS_FULLPATH_BOOST}/include
                         ${IRODS_EXTERNALS_FULLPATH_FMT}/include
+                        /opt/irods-externals/spdlog1.5.0-1/include
```

Comment:
4-2-stable does not have spdlog as an externals package.

check if the IRODS_EXTERNALS_PREFIX exists.

Reply:
we'll need a spdlog externals pkg for 4-2-stable